Add July 2025 newsletter (#2395)
writethedocs/www: docs/blog/newsletter-july-2025.rst

Several people suggested some best practices to consider. In general, instead of using animated GIFs, use short videos (MP4 or WebM files) with user controls. (Note: You can convert animated GIFs to such files.) If you *must* use animated GIFs with no controls, limit them to very short and purposeful animations. Also consider showing a static GIF with instructions that the user can click to view the animated version.

See more Write the Docs resources about `media </topics/#other-media>`__.

----------------------------------------------
Strategies for successful documentation audits
----------------------------------------------

> For your next documentation audit, consider these ideas to keep it organized and help make sure that it's a success.
> 
> Start by clarifying your goals. You might only want to map your content, or you might want to improve it. You might know specific issues you need to fix, or you might not be sure what problems you'll uncover. Think about how ambitious your audit should be and how much time you can dedicate.

Author: ```suggestion
For your next documentation audit, consider these ideas to keep it organized and help make sure that it's a success.

Start by clarifying your goals. You might only want to map your content, or you might want to improve it. You might know specific issues you need to fix, or you might be trying to uncover them.. Think about how ambitious your audit should be and how much time you can dedicate.
```
